### ISSUE TITLE
No more client revocation feature

### ISSUE THREAD

**Issue by ulvida** [Stouts/Stouts.openvpn#160]
The [variable `openvpn_clients_revoke`](https://github.com/Stouts/Stouts.openvpn/blob/develop/defaults/main.yml#L119) is no longer used in the role. 

Indeed, it is now in the scope  of the separate easy_rsa role to deal with clients credentials. 

However, the openvpn role should at least manage the deletion of [the .ovpn file it creates here](https://github.com/Stouts/Stouts.openvpn/blob/f51f20c62be415282bf43e834010cebe979ab2cb/tasks/core/clients.yml#L3). 

We're including this in a PR soon.

**Comment by nkakouros** [comment]
Hm, indeed. Please try to break the PR into meaningful chunks. There is one already [open for ipv6 support](https://github.com/Stouts/Stouts.openvpn/pull/151), perhaps you want to take a look at it too.

**Comment by ulvida** [comment]
Yes, when I say "we" I mean [our team at the UdelaR](https://github.com/UdelaRInterior), who are working on [this fork of your role](https://github.com/UdelaRInterior/Stouts.openvpn/). So I speak about the same PR, sorry. 

I'm putting issues here but [PRs on our fork](https://github.com/UdelaRInterior/Stouts.openvpn/pull/4), saying in commits which issues they will close. @santiagomr is checking and integrating all in a PR with clear meaningful chunks.
